src/components/TransactionsInterpreter/helpers/getDenominatedValue.ts

```diff
@@ -0,0 +1,23 @@
+import { UITransactionType } from './types';
+import { networkConfigSelector } from 'reduxStore/selectors';
+import { store } from 'reduxStore/store';
+import { denominate } from 'utils';
+import { DenominationConfig } from '../TransactionsInterpreter';
+
+export function getDenominatedValue(
+  transaction: UITransactionType,
+  { decimals, denomination, showLastNonZeroDecimal = false }: DenominationConfig
+) {
+  const networkConfig = networkConfigSelector(store.getState());
```

Comment:
This call makes this function depend on global state. What if an app wants to use dapp-core as an utils library, in a React Native application? Maybe we should make this passable from outside, the default stuff, or just fallback to something that's hardcoded in constants